src/modules/starboard/starboard.ts

```diff
@@ -8,6 +9,52 @@ import {
 import { StarboardMessage } from "../../store/models/StarboardMessage.js";
 import { createStandardEmbed } from "../../util/embeds.js";
 import { config } from "../../Config.js";
+import ffmpeg from "fluent-ffmpeg";
```

Comment:
It seems like this library is abandoned. It's not a huge issue necessarily, but it might be easier to use something like https://github.com/mmomtchev/ffmpeg